superset-frontend/packages/superset-ui-core/src/utils/html.tsx

```diff
@@ -28,7 +28,7 @@ export function sanitizeHtml(htmlString: string) {
 }
 
 export function isProbablyHTML(text: string) {
-  return /<[^>]+>/.test(text);
+  return /<\/?[a-z][\s\S]*>/i.test(text);
```

Comment:
@justinpark is there merit in using a non-regex implementation, i.e., something like [this](https://stackoverflow.com/a/55680987)?

Reply:
that looks good.let me try with the alternative solution.